### PR TITLE
Update AVI to AVAM to reflect new service name

### PR DESCRIPTION
**Pull request by shanepeckham**

### CODE REVIEW

File: VideoSearchWithAutoMLVision/README.md [hunk 1]
```diff
@@ -124,7 +124,7 @@ terraform apply
 
 #### Expected result
 
-After Terraform reports that the apply is complete, if you navigate to your resource group, having used the default
+After Terraform reports that the apply is complete, if you nAVAMgate to your resource group, hAVAMng used the default
```
Comment (ika-microsoft) [inline VideoSearchWithAutoMLVision/README.md:127]: Typo

File: VideoSearchWithAutoMLVision/README.md [hunk 2]
```diff
@@ -144,41 +144,41 @@ used.
...
-#### *Video Indexer Account*
+#### *Azure Video Analyzer for Media Account*
 
-First, we'll need to set up a Video Indexer account linked to the Media Services Resource deployed (name prefix
+First, we'll need to set up a Azure Video Analyzer for Media account linked to the Media Services Resource deployed (name prefix
 `mediaservices`).
 
-If we navigate to this resource's overview, and scroll down, there is a window entitled "Video Indexer". Click the link
-"[Go to Video Indexer](https://www.videoindexer.ai/)".
+If we nAVAMgate to this resource's overview, and scroll down, there is a window entitled "Azure Video Analyzer for Media". Click the link
```
Comment (ika-microsoft) [inline VideoSearchWithAutoMLVision/README.md:152]: Here same

File: VideoSearchWithAutoMLVision/README.md [hunk 3]
```diff
@@ -144,41 +144,41 @@ used.
...
 When creating the new account, select the Azure subscription and Media Services account (name prefix `mediaservices`)
 used in the resource deployment.
 
 If you see an error displayed after the account has been created, saying `There’s a problem with your connection to Azure Media Services. Update connection.`, ignore it.
 
-Once created, go to the [Video Indexer developer portal](https://api-portal.videoindexer.ai/).
+Once created, go to the [Azure Video Analyzer for Media developer portal](https://api-portal.videoindexer.ai/).
 
-Sign in using the same AAD account, and go to "Profile" in the navigation bar.
+Sign in using the same AAD account, and go to "Profile" in the nAVAMgation bar.
```
Comment (ika-microsoft) [inline VideoSearchWithAutoMLVision/README.md:169]: Same here. Please fix all over the place

**Review by ika-microsoft — CHANGES_REQUESTED** [review]
There are many wrong search and replaces.

**Review by tshaiman — APPROVED** [review]
LGTM

**Pull request merged**